packages/react-components/react-tags/stories/TagGroup/TagGroupBestPractices.md

```diff
@@ -2,4 +2,8 @@
 
 ### Do
 
+- When using `TagGroup` with `Combobox` to display multiple selected values:
```

Comment:
I think it'd also be good to mention here that the Picker component is planned, and will be the recommended approach once it's available

Reply:
Updated :)